src/stare/models/analysis.py

```
@@ -1,68 +1,127 @@
+    dataset_used: str | None = None
+    model_tested: str | None = None
+    methods: str | None = None
+    editorial_board_formed_on: date | None = None
+    pgc_or_sgc_sign_off_date: date | None = None
+    analysis_contacts: list[AnalysisContact] = Field(default_factory=list)
+    editorial_board: list[EditorialBoardMember] = Field(default_factory=list)
+    meetings: list[TypedMeeting] = Field(default_factory=list)
+
+    @model_validator(mode="before")
+    @classmethod
+    def _flatten_meetings(cls, data: Any) -> Any:
```

Ruff (0.15.10)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `data`

(ANN401)